content/docs/manuals/networking-internals/ipv6-addressing.md

```diff
@@ -9,4 +9,64 @@ seo:
+| `fdfe::/16`                                             | reserved for future use            | `reserved` |
+| `fdff::/16`                                             | Service IP subnet                  | `service`  |
+| `fdff:0000::/21`                                        | Instance IP subnet 1               | `instance` |
+| `fdff:1000::/21`                                        | reserved for Closest balancing policy | `balance`  |
+| `fdff:2000::/21`                                        | Round Robin balancing              | `balance`  |
+| ...                                                     | ...                                | `balance`  |
+| `fdff:f000::/21`                                        | Balancing 15                       | `balance`  |
+| `fdff:0800::/21`                                        | Instance IP subnet 2               | `instance` |
+| `fdff:1800::/21`                                        | Balancing 16                       | `balance`  |
+| `fdff:2800::/21`                                        | Balancing 17                       | `balance`  |
+| ...                                                     | ...                                | `balance`  |
+| `fdff:f800::/21`                                        | Balancing 30                       | `balance`  |
+
+In the following we give a short meaning for each tag.
+
+* `all` : whole Oakestra platform
+* `worker` : subnet assigned to a single worker node associated with a cluster inside Oakestra
+* `service` : semantic service address subnet
+* `instance` : single semantic service address mapped to a service instance throughout the lifetime of the instance
+* `balance` : semantic service address, addressing all multiple instance addresses of a service in order to enable
+semantic addressing for balance policies
+
+{{< callout context="danger" title="Exhausted Address Space" icon="outline/alert-triangle" >}}
+The Oakestra platform makes use of the whole IPv6 address space available.
+This may cause issues with other applications requiring an IPv6 address from the `fc00::/7`
+address space in order to function, as we make changes to the `iptables` local firewall,
+as well as the host routing tables, that may influence any third-party application's functionality.
+{{< /callout >}}
+
+## Example Application
+
+If we deploy an example application A with Service S and two instances (S1 and S2) on the same worker node W,
+an example address configuration can be the following:
+
+| App A    | Addresses / Subnets assigned                                |
+|----------|-------------------------------------------------------------|
+| Worker W | `fc00::c0:ffee:/120`                                        |
+| S        | Round Robin: `fdff:2000::30`                                |
+| S1       | Instance IP: `fdff::2f`<br> Namespace IP: `fc00::c0:ffee:2` |
+| S2       | Instance IP: `fdff::30`<br> Namespace IP: `fc00::c0:ffee:3` | 
+
+
```

Comment:
Again add how Ipv6 address space can be customized

Reply:
done in latest commit